### PR TITLE
Allow external registry name used by clients to diverge from internal name for connections when receiving events

### PR DESCRIPTION
**Pull request by jmahowald**
Description from https://github.com/SUSE/Portus/issues/579#issuecomment-220350888

Also, I know I have no spec tests yet.  My rails skills aren't that great, but I'm happy to add in.  Wanted first to see whether you approve the high level changes

### CODE REVIEW

File: config/config-local.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  filter: ""
+
+  # The LDAP attribute where to search for username. The default is 'uid'.
+  uid: "uid"
+
+  # LDAP credentials used to search for a user.
+  authentication:
+    enabled: true
+    bind_dn: "CN=dockerregistry_srv,OU=Accounts System,DC=us,DC=int,DC=genesyslab,DC=com"
+    password: "8L2SF5hJ8BWq8tsbxrSqBzuKpUetdf"
```
Comment (cyphar) [inline config/config-local.yml]: I would change this password if I were you. ;)

Comment (jmahowald) [inline config/config-local.yml]: Doh.   Thanks, certainly hadn't meant for that file to be in.

Comment (mssola) [inline config/config-local.yml]: Yeah, just ignore the whole `config-local.yml` file altogether :wink:

**Comment by mssola** [comment]
This is something I certainly didn't think about, thanks :+1: In principle it looks good to me. Just ignore the `config-local.yml` file and provide tests :wink: 

@flavio what do you think ?

**Comment by flavio** [comment]
Sorry for the late response. @jmahowald could you provide some background about this feature. What use case are you trying to solve?

**Comment by mssola** [comment]
Hey @jmahowald, are you still working on this ? Could you rebase your patch ?

**Comment by jmahowald** [comment]
Hey, sorry.  I can work to close this out, but the reason I haven’t done as much is because we decided to roll with gitlab which just added in support for registries and we already have deployed and integrated with our corp AD

**Comment by mssola** [comment]
Closing then. I don't have any objections on this idea (as long as it doesn't over-complicate the setup for regular use cases).  So, if anyone wants to pick it up again and submit a new PR, feel free to do so.

**Pull request closed without merging**